docs/src/components/Popup.css

```diff
@@ -1,7 +1,7 @@
 @layer components {
   .Popup {
-    max-width: var(--available-width);
-    max-height: var(--available-height);
+    max-width: var(--popover-available-width);
+    max-height: var(--popover-available-height);
```

Comment:
If we add component prefixes, this makes classes like this non-reusable across different popups... @vladmoroz. In fact, I'm not sure if this popup is used for popover specifically.

Reply:
Hmm yeah good point, I was thinking that the Popup component would be used across different popups, such as Popovers, Preview Cards, Dialogs, etc. Let's discuss it on Monday in detail. We need to consider whether prefixing everything maps well to how components would be styled.